src/controls/viewPicker/IViewPicker.ts

```diff
@@ -0,0 +1,100 @@
+import { BaseComponentContext } from '@microsoft/sp-component-base';
+import { ISPView } from "../../../src/common/SPEntities";
```

Comment:
Please, use the same quotes style: either `'` or `"`

Reply:
Hello @AJIXuMuK 

Updated as suggested.

Thanks,
Nishkalank